docs/howto/upgrade-cluster/aws.md

```diff
@@ -2,74 +2,17 @@
 
 # Upgrade Kubernetes cluster on AWS
 
-```{warning}
-This upgrade will cause disruptions for users and trigger alerts for
-[](uptime-checks). To help other engineers, communicate that your are starting a
-cluster upgrade in the `#maintenance-notices` Slack channel and setup a [snooze](uptime-checks:snoozes)
```

Comment:
We shouldn't need a snooze, so that was removed. I relocated the note on communicating in slack to a dedicated step.